.markdownlint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "MD012": {
+      "maximum": 2
+  },
+  "MD013": false
+}
```

Comment:
What is this? Who checks these rules?

Reply:
This is the configuration for markdownlint in VS Code. Should I remove it? (It allows two consecutive blank lines and disables line length limits)

Reply:
If this is editor specific, I don't think it should be part of the repository. If this is a general tool with github plugin support, just add a comment what it is. From your explanation these rules sound like a use case for a general linter.